js_modules/dagit/packages/core/src/pipelines/PipelineExplorerRoot.tsx

```diff
@@ -28,7 +28,7 @@ import {
 } from './types/PipelineExplorerRootQuery';
 
 export const PipelineExplorerRegexRoot: React.FC<
-  RouteComponentProps & {repoAddress: RepoAddress}
+  RouteComponentProps & {repoAddress: RepoAddress; pageContext: 'graph' | 'job'}
```

Comment:
I might be wrong, but it looks like `PipelineExplorerRegexRoot` is only used by `GraphRoot`, so there's not a scenario where this would be `job`?  I think we could probably move this component and rename it GraphRegexRoot and we just didn't to keep the original diff smaller?

tbh I'm having a hard time jumping back in to this particular code path, the fact that pipeline <> job are still sprinkled around is rough 😬  Will be nice to clean this up soon!